ext/skeleton/skeleton.c

```diff
@@ -71,7 +71,7 @@ PHP_MINFO_FUNCTION(%EXTNAME%)
  */
 static const zend_function_entry %EXTNAME%_functions[] = {
 	PHP_FE(test1,		arginfo_test1)
-	PHP_FE(test2,		arginfo_test2)
+	PHP_FE(%EXTNAME%_test2,		arginfo_test2)
```

Comment:
This should be the other way around, the `%EXTNAME%_` on the declaration should be dropped to make things line up.

Reply:
Oh, in line 31 Máté Kocsis forgot to drop the `%EXTNAME%_`
```c
PHP_FUNCTION(%EXTNAME%_test2)
```